### DISCUSSION TITLE
希望能在普通记本的回应里面增加添加图片的功能选项

### DISCUSSION THREAD

**Discussion by Airthink** [WatsonYao/nian_app#92] [Q&A] [answered]
比较希望有这个功能，我个人一般比较喜欢记录在网上知道的东西，大部分学习到了之后会有一个反馈，但是反馈的话一般都需要截屏来回应一下，顺带继续增加记录，回应的话没有增加图片的一个选项。所以我希望能增加一个这个功能，您看看可以考虑一下。

顺带希望能够在普通记本的图片中增加可以缩放的功能。

**Comment by WatsonYao** [comment, marked as answer]
回应不再添加更多功能，回应也不会参与统计相关的计算，建议多用进展本身。
另外，nian没有集成任何图片附加功能，所以大图缩放等，都不会支持，全部需要跳转第三方操作。
